### ISSUE TITLE
The project file 'D:\C# Projects\Game1\Game1\Game1.csproj' cannot be opened. There is a missing project subtype.

### ISSUE THREAD

**Issue by geejayz** [MonoGame/MonoGame#6082]
I have VS2017 Community installed.
I just installed MonoGame 3.6 and was going to try one of the tutorials.
When I try to create a new project with template "MonoGame Windows Store (XAML) Project" I get an error of;

The project file 'D:\C# Projects\Game1\Game1\Game1.csproj' cannot be opened. There is a missing project subtype....

I note on the website that it doesn't mention support for VS2017 and wonder is this the issue?

I also note that in the error message there would appear to be a reference to a Game1 subfolder inside another Game1 folder which seems incorrect.

Are there other dependencies that I need to install or do I need to work with an earlier version of VS?

**Comment by tomspilman** [comment]
> I just installed MonoGame 3.6

This should be fixed in 3.7... please install the latest development version of the MG installer and try it again.

http://teamcity.monogame.net/repository/download/MonoGame_PackagingWindows/latest.lastSuccessful/MonoGameSetup.exe?guest=1

**Comment by geejayz** [comment]
Thanks for your feedback Tom, I appreciate your help, I'll hopefully get a chance to try this again this evening (or worst case tomorrow).

**Comment by DDReaper** [comment]
I noted this in a few talks and my Getting Started with MG on VS 2015/2017.

For 2017, you need to be using the latest beta on the downloads page as @tomspilman noted above.
You can check the getting started videos here:
https://www.youtube.com/watch?v=zphaylhOrm0

**Comment by geejayz** [comment]
Hi Simon - I used your video to check my VS2017 install and can confirm that all the suggested inclusions are there. 

Unfortunately, despite this and installing from the Repo advised by Tom, I'm still getting the same error on trying to create a new project.

MonoGame Install Version: 3.7.0.1111
VS2017 v15.4.5

**Comment by DDReaper** [comment]
A daft thing to try, could you create the project in a new root folder, say d:\development ?
Just to see if there is something funny going on with that folder structure.

If you are using the latest 3.7 build, also make sure to try both of the Universal Templates.

In short though, if you have UWP development installed with the relevant UWP SDK's, that should be all you need.

**Comment by geejayz** [comment]
I did a reinstall of VS2017 and also tried letting the project create at the default location in C: but I still get the same error.

I tried the Windows 10 XAML option and it opened the settings control panel. I 'applied' what appear to be suggested changes to the settings and closed it. It looked like it hung trying to create the project but eventually completed about 1 minute later. Looks like the Windows 10 option works OK.

That's probably enough to get me started (assuming the beginner tutorials will work with either of those templates.

Thanks guys for the help with this.
